test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/LoadingSample.cs

```diff
@@ -6,23 +6,64 @@ namespace Tizen.NUI.Samples
 {
     public class LoadingSample : IExample
     {
-        private TextLabel board1, board2, board;
-        private Button button1, button2, button3, button4;
-        private Loading loading1_1, loading2_1;  //1-null para 2-attributes; X_1-color; X_2 image track
+        private TextLabel propBoard, propLogBoard, attrBoard;
+        private Button propFpsAddBtn, propFpsMinusBtn, attrBtn;
+        private Loading propLoading, attrLoading;
         private View root;
+        private View parent, propParent, propBtnParent, attrParent;
+        private string[] imageArray;
 
         public void Activate()
         {
             Window window = NUIApplication.GetDefaultWindow();
 
             root = new View()
             {
-                Size2D = new Size2D(1920, 1080),
+                Size = new Size(1920, 1080),
             };
+            window.Add(root);
+
+            parent = new View()
+            {
+                Position = new Position(100, 200),
+                Size = new Size(1000, 780)
+            };
+            parent.Layout = new LinearLayout()
```

Comment:
If it is kind of Grid, we have grid layout

Reply:
Used gridLayout